pepper-apis/src/main/java/org/broadinstitute/ddp/service/KitCheckService.java

```diff
@@ -133,6 +133,7 @@ public KitCheckResult checkForInitialKits(Handle apisHandle) {
      * @param apisHandle the database handle
      * @param dsmClient  the dsm client
      */
+    @Deprecated
```

Comment:
Remove unused code/functionality will remove a lot of code. Trying to keep PR digestible, so I'll remove those deprecated items in a later PR.